src/services/services.ts

```diff
@@ -1168,7 +1171,7 @@ module ts {
     }
 
     export interface Classifier {
-        getClassificationsForLine(text: string, lexState: EndOfLineState, classifyKeywordsInGenerics?: boolean): ClassificationResult;
+        getClassificationsForLine(text: string, lexState: EndOfLineState, syntacticClassifierAbsent?: boolean): ClassificationResult;

```

Comment:
Comment the purpose of this parameter so that consumers of the LS know what they should pass and how hte classifiers interact